### ISSUE TITLE
Plant Bioacoustics Syllabus

### ISSUE THREAD

**Issue by MarkBruns** [Botanical-Cloud/AwesomeBotanicals#3]
O'Reilly Playlist

[Plant bioacoustics](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C14&q=Plant+bioacoustics&btnG=) on [Google Scholar](https://scholar.google.com/)

**Comment by Knappen** [comment]
this subject is a pretty large mass to grope and extremely interesting. there were researches done on intuition and somewhat supported by verified results decades ago on the effects of sound on the plants.

one dude (i think he was an federal ag head from Iowa) set up high pitched tweet sounds in a field in southeast asia that made some improvements in yield. another (i think it was Baxter) set up plants in a chamber with different music. the plants exposed to acid rock whithered. the plants exposed to BEEthoven flurished.

not sure why i'm supprised science has been taken this subject seriously; finally. but it's good to know it's getting done.

**Comment by MarkBruns** [comment]
Agreed!  This topic of ultrasonic bioacoustics moves beyond just being an intriguing curiosity and becomes a very serious research discipline in a hurry ... and still a relatively new discipline that has not yet received 6attention it deserves.  

SERIOUS ultrasonic acouustic analysis [with a DAW and AI/ML automation, because there'll be so incredibly data to analyze] begins with LEARNING THE BASICS ... ultrasound bioacoustics and other aspects of physics, digital audio workstations. configuration management, distributed version control. AI/ML and deep learning beyond BERT and Jupyter notebooks [with audio analysis/visualiziation libraries]
